auth/src/main/java/com/firebase/ui/auth/ui/AcquireEmailHelper.java

```diff
@@ -80,30 +80,22 @@ private void startEmailHandler(String email, List<String> providers) {
             mActivityHelper.startActivityForResult(registerIntent, RC_REGISTER_ACCOUNT);
         } else {
             // account does exist
-            for (String provider : providers) {
```

Comment:
Can we be sure that if the email provider is in the `providers` collection that it will be at index `0`?  I agree the loop was not needed but not sure that `get(0)` will always be equivalent.